src/wallet/api/walletServiceAxios.ts

```diff
@@ -26,6 +26,12 @@ export const axiosInstance = async (wallet: HathorWalletServiceWallet | null = n
   const defaultOptions = {
     baseURL: WALLET_SERVICE_BASE_URL,
     timeout: timeout,
+    // `validateStatus` defines whether to resolve or reject the promise for a given
+    // HTTP response status code. If `validateStatus` returns `true` (or is set to `null`
+    // or `undefined`), the promise will be resolved; otherwise, the promise will be
+    // rejected. The default behaviour of axios is to reject anything different than 2xx
+    // We need to handle some 400 manually (e.g. create wallet might already be loaded)
+    validateStatus: (status) => status >= 200 && status < 500,
```

Comment:
I'm not sure I agree. We want to be able to handle some errors accordingly with what we have in the wallet service, e.g. the 400 error in the load wallet that we want to manually handle it. Why should we leave axios to reject it if we need to handle them?

Reply:
Can't you use something like this? I mean, I don't think we should change the default behavior because just one API needs a different behavior. https://github.com/axios/axios#handling-errors

Reply:
Yeah, makes sense. Done here https://github.com/HathorNetwork/hathor-wallet-lib/pull/221/commits/ac0b4b4d5f9f57d0aedff47e5d4f18baf3b7a00b

Reply:
Great! Thanks!